css-layout-classes/styles.css

```diff
@@ -0,0 +1,35 @@
+#main-container {
+  padding: 0 100px;
+  font-family:
+    Arial,
+    Helvetica,
+    sans-serif;
+  font-size: large;
+}
+
+.main-head {
+  font-size: xx-large;
```

Comment:
I would avoid using font-sizes that aren't rem, em, or px sizes. its much easier to make adjustments later when you have a set value easy to recognize.